DynamicProgramming/dice_throw_problem

```diff
@@ -0,0 +1,30 @@
+// Java program to find number of ways to get sum 'x' with 'n'  
```

Comment:
Your file should also have a  . java extension. I am closing the pr kindly make the changes and reopen it.